app/models/session_notification.rb

```diff
@@ -84,12 +86,27 @@ def self.create_and_send!(
 
     parents.each do |parent|
       params = { parent:, patient:, session:, sent_by: current_user }
+      template_name = compute_template_name(type, session.organisation)
 
-      EmailDeliveryJob.perform_later(:"session_#{type}", **params)
+      EmailDeliveryJob.perform_later(template_name, **params)
 
       next if type == :school_reminder && !parent.phone_receive_updates
 
-      SMSDeliveryJob.perform_later(:"session_#{type}", **params)
+      SMSDeliveryJob.perform_later(template_name, **params)
+    end
+  end
+
+  def self.compute_template_name(type, organisation)
+    template_name = "session_#{type}"
+
+    if organisation.ods_code == CWPT_ODS_CODE
+      cwpt_template_name = "cwpt_#{template_name}".to_sym
+
+      if GOVUK_NOTIFY_EMAIL_TEMPLATES.key?(cwpt_template_name)
+        return cwpt_template_name
+      end
```

Comment:
I'm wondering if an approach where we add the ODS code as a suffix and see if that template exists would work better:

```rb
template_names = [:"session_#{type}_#{organisation.ods_code}", :"session_#{type}"]

template_name = template_names.select { GOVUK_NOTIFY_EMAIL_TEMPLATES.key?(it) }.first
```

Then that would mean we don't need to hard code Coventry's ODS code in this file, and it would allow us to easily have the option of custom templates per organisation in the future if we needed it (although we hopefully won't).

Reply:
Good idea. 👍